### PR TITLE
TNW-Clone

### PR DESCRIPTION
**Pull request by jstloyal**
What does this PR do?

Shows how to use media queries to make a website responsive.

Description of Task to be completed?

Added the HTML index file
Added CSS to style and position the element.
Ran validations on our Html and CSS.

How should this be manually tested?
Clone this branch.
open the index.html file on your browser

Screenshot
TNW-Clone

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+}
+
+html {
+  font-size: 62.5%;
+}
+
+html,
+body {
+  max-width: 144rem;
+}
```
Comment (majovanilla) [inline css/main.css]: This is not the correct width. Right now your page looks smaller in width (it should occupy the whole screen). Ii should be centered, too, and it's not. Please adjust those properties to your body tag.
Screenshot from 2020-03-18 11-51-06

File: index.html [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>  
+        </nav>
+        <main>
+            <div class="fixed-nav">
+                <div class="bottom-nav">
+                    <div class="tnw-logo">
+                        <img alt="TNW-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+                    </div>
```
Comment (majovanilla) [inline index.html]: - [ ] The logo shouldn't appear in the large breakpoint of your page.
Screenshot from 2020-03-18 11-53-28

Original:
Screenshot from 2020-03-18 11-53-34

File: index.html [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+                        <div class="time-author">
+                          <p>by Satoshi Nakaboto</p>
+                          <p><i class="clock outline icon"></i>2 hrs ago</p>
+                          <p><i class="share alternate icon"></i>5</p>
+                        </div>
+                    </article>
+                </div>
+
+                <section>
+                    <h2>Latest New <i class="chevron right icon"></i></h2>
```
Comment (majovanilla) [inline index.html]: - [ ] The latest news section title should be aligned with the articles' images (both medium and large screens) and should be red.
Screenshot from 2020-03-18 12-01-19

Original:
Screenshot from 2020-03-18 12-01-33

- [ ] In the mobile version, your page has larger images. Please fix them.
Screenshot from 2020-03-18 12-07-13

Original:
Screenshot from 2020-03-18 12-07-06

File: index.html [hunk 4]
```diff
@@ -0,0 +1,555 @@
...
+                    <div class="my-flex">
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/ajwjl6u-540x270.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>VW Wants its EVs to become part of the power grid when charging.</p>
+                                <small>Mathew Beedham <i class="fa fa-circle"></i>7 minutes ago</small>
+                            </div>
```
Comment (majovanilla) [inline index.html]: - [ ] All the additional-content divs should have a link inside, not a `p` element. The text should be bold. Please change it in all the section's articles.

- [ ] The `small`  tag text should be gray and have a link in the author's name.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,555 @@
...
+                                <p> $30M </p>
+                                <small>SERIES B</small>
+                            </div>
+                            <div class="fund2">
+                                <p><i class="tag icon"></i> E-commerce, +2</p>
+                                <p><i class="tag icon"></i> Rakuten, +5 </p>
+                            </div>
+                            <div class="fund3">
+                                <small> SG's Cash back App Shopback closes $75M Series B led by Tamasek On Dealstreetasia </small>
+                            </div>
```
Comment (majovanilla) [inline index.html]: - [ ] Check the original width and padding of the elements in this section, yours look less wide and have a lot of padding.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,555 @@
...
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/08/Discord-Go-Live-516x270.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>White House asks Silicon Valley for AI solutions to Corona Virus</p>
+                                <small>Thomas Macaulay <i class="fa fa-circle"></i>14 hours ago</small>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <div class="funding">
```
Comment (majovanilla) [inline index.html]: - [ ] In the mobile version, you should only show three articles in this section. Yours has four. Please adjust this. 

- [ ] The section's articles should have 100% width when mobile. Please make this change.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,555 @@
...
+                            <div class="fund3">
+                                <small> Harbour Biomed raises $67.4M to treat Corona Virus and Cancer on Silicon Canals - European technology News </small>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="tech-news">
+                    <div class="apps-info">
+                        <h1>Apps <i class="chevron right icon"></i></h1>
+                        <div class="app-content">
```
Comment (majovanilla) [inline index.html]: - [ ] All the articles in this section should have a larger image when in mobile screen size. Please adjust it.
Screenshot from 2020-03-18 12-12-42

Original:
Screenshot from 2020-03-18 12-12-35


- [ ] Check the original width and padding of the elements in this section, yours look less wide and they should occupy all the width of the screen.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,555 @@
...
+                    </div>
+                </div>
+
+                <section class="last-section">
+                    <header>
+                        <h2>Latest Deals <i class="chevron right icon"></i></h2>
+                        <p>See all Deals</p>
+                    </header>
+                    <div class="deals-flex">
+                        <div class="deals-item">
```
Comment (majovanilla) [inline index.html]: - [x] The elements in this section have a large image when in mobile screen size. 

- [x] The title is not aligned correctly.

Screenshot from 2020-03-18 12-16-16

Original;
Screenshot from 2020-03-18 12-16-10

File: index.html [hunk 9]
```diff
@@ -0,0 +1,555 @@
...
+                <a href=""><i class="rss icon"></i></a>
+                <a href=""><i class="envelope outline icon"></i></a>
+            </div>
+            <div class="footer-info">
+                <p>EVENTS</p>
+                <p>ABOUT</p>
+                <p>ADVERTISE</p>
+                <p>JOBS</p>
+                <p>CONTACTS</p>
+            </div>
```
Comment (majovanilla) [inline index.html]: - [x] These elements should be links.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,555 @@
...
+            <p class="bottom-footer">
+              <img class="tnw-footer" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="tnw-logo"/> &COPY; 2006 - 2020 The Next Web B.V. Made with <i id="heart" class="heart icon"></i> in Amsterdam. 
+              Powered By <img class="stackpath" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/stackpath.svg" alt="stackpath" />.
+              </p>
+            <div class="bottom-mobile-footer">
+                <p><img class="tnw-footer" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="tnw-logo"/> &COPY; 2006 - 2020 The Next Web B.V.</p>
+                <p>Made with <i id="heart-mobile" class="heart icon"></i> in Amsterdam.</p>
+                <p> Powered By <img class="stackpath" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/stackpath.svg" alt="stackpath" />.</p>
+            </div>
```
Comment (majovanilla) [inline index.html]: - [x] The small logos should be links.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,555 @@
...
+                      
+               </div>
+               <p class="p1"><a href="#">Did you know we have a TECH FESTIVAL? Check it out <i class="long arrow alternate right icon"></i></a></p>
+           </div>
+            
+            <div class="main-grid">
+                <div class="tnw">
+                    <div>
+                        <img src="./images/tnw-w.png" alt="tnw-logo"/>
+                        <time datetime="2020-03-13 20:00">Friday - March 13, 2020</time>
```
Comment (majovanilla) [inline index.html]: - [ ] You date is overlapping the rest of the elements when you change the screen size. Please fix this.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey team, 👋

## Good job on this project! 👍
- Good Readme file
- Good Pull Request Message

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- [ ] Always remember to check your HTML using the [validator](https://validator.w3.org/). You have some errors and warnings. Please fix them.

- [ ] Your sections have a lot of space between each other and that's not the case of the original page, please make them look like the source.

- [ ] Check your fonts and adjust the weight to match the titles (they should be bold) in most of your sections and articles' titles.

- [ ] See more comments below and make the required changes.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Your project would look sooo much better if you include the hover effects on the links of the page, like the original version.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by ghost** [comment]
Fixed all the errors.

**Review by rOluochKe — APPROVED** [review]
Hello Team
### Your project has been approved. :boom: :100: Congratulation.
### You can merge and submit the project from your dashboard. Good luck with the next project.
- Happy coding!

**Pull request merged**